frontend/pages/user/profile/edit.vue

```diff
@@ -108,14 +108,14 @@
         :label="$t('profile.show-advanced-description')"
         @change="updateUser"
       ></v-checkbox>
+      <a class="mt-5 d-flex flex-column justify-center text-center" to="/group"> {{ $t('profile.looking-for-privacy-settings') }} </a>
```

Comment:
Why did you change this to an a tag? I believe it should be a Nuxt Link https://nuxtjs.ir/api/components-nuxt-link